editor/src/components/canvas/controls/size-box.tsx

```diff
@@ -232,6 +325,34 @@ const ResizeLines = (props: ResizeLinesProps) => {
         edge={edge}
         color={props.color}
       />
+      {(showLabel || isEdgeDragged) && (
```

Comment:
`when(`

Reply:
Fixed in 5b68dd64a689f218fbe4e203d4db7b5a9433a636.